src/components/styled-components/StyledInput.js

```diff
@@ -45,7 +45,22 @@ const StyledInput = styled.input`
 		height: 2.7rem;
 		margin-left: 1rem;
 	
-`}
+	`}
+
+	${({ variant }) =>
+		variant === 'complexity' &&
+		`
```

Comment:
```suggestion
		css`
```
if you import this helper function from styled-components the template string gets displayed as css code again.